src/app/static/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "author": "",
   "license": "MIT",
+  "web-types": "./web-types.json",
```

Comment:
This is used by intellij to document the v-translate directive, so that I no longer have a warning that this doesn't exist in my IDE. See https://github.com/JetBrains/web-types

Let me know if that is a problem for vscode?